src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -985,13 +998,13 @@ public virtual Task<IdentityResult> RemoveClaimAsync(TUser user, Claim claim)
             var claimStore = GetClaimStore();
             if (user == null)
             {
-                throw new ArgumentNullException("user");
+                throw new ArgumentNullException(nameof(user));
             }
             if (claims == null)
             {
-                throw new ArgumentNullException("claims");
+                throw new ArgumentNullException(nameof(claims));
             }
-
+            
```

Comment:
Can you autoformat the all the files as well?  It looks like you added a bunch of misc white space in a bunch of places